addons/website_slides/models/slide_channel.py

```diff
@@ -153,7 +154,11 @@ def _default_access_token(self):
     karma_gen_slide_vote = fields.Integer(string='Lesson voted', default=1)
     karma_gen_channel_rank = fields.Integer(string='Course ranked', default=5)
     karma_gen_channel_finish = fields.Integer(string='Course finished', default=10)
-    # TODO DBE : Add karma based action rules (like in forum)
+    # Karma based actions
+    karma_review = fields.Integer('Add a review', default=10, help="Karma needed to add a review on the course")
+    karma_slide_comment = fields.Integer('Add a comment', default=3, help="Karma needed to add a comment on a slide of this course")
+    karma_slide_vote = fields.Integer('Vote on slide', default=3, help="Karma needed to like/dislike a slide of this course.")
+    can_review = fields.Boolean('Can Review', compute='_compute_karma_rights')
```

Comment:
_compute_can_review

Reply:
For the love of the flemme, even if it's not guideline compliant, let's keep `_compute_karma_rights` as it's the same in forum ;)